timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -16,6 +16,7 @@
     TransformerModel,
 )
 from .sktime import ARIMAModel, AutoARIMAModel, AutoETSModel
```

Comment:
if all goes well, can we use this PR to also add guards for `sktime` and dropping it from the hard requirements?

Reply:
How about doing this in a separate PR? I think this PR will already be quite large after all the TODOs are completed